nglview/show.py

```diff
@@ -281,6 +286,29 @@ def show_schrodinger_structure(mol, **kwargs):
     return NGLWidget(structure_trajectory, **kwargs)
 
 
+def show_biopython(mol, **kwargs):
+    '''Show NGL widget with Biopython structural entity.
```

Comment:
you need to add biopython to

either

- pip: https://github.com/arose/nglview/blob/master/pip-requirements-test.txt
- conda: https://github.com/arose/nglview/blob/master/conda-requirements-test.sh